c7n/resources/appelb.py

```diff
@@ -255,6 +256,8 @@ def process(self, resources, event=None):
 class WafV2Enabled(Filter):
     """Filter Application LoadBalancer by wafv2 web-acl
 
+    Supports regex expression for web-acl
```

Comment:
@cahn1 Might be useful to add a blurb about how this can be helpful with WAFs that are managed by AWS Firewall Manager

Reply:
Added a blurb